src/main/java/dev/jbang/cli/Edit.java

```diff
@@ -168,7 +171,23 @@ private boolean openEditor(File project, String projectPathString) throws IOExce
 				cmd = new String[] { "cmd", "/c", editorCommand };
 			}
 			verboseMsg("Running `" + String.join(" ", cmd) + "`");
-			new ProcessBuilder(cmd).start();
+			ProcessBuilder pb = new ProcessBuilder(cmd);
+			Path javaHome = JdkManager.getCurrentJdk(ctx.getJavaVersionOr(ss));
+			if (javaHome == null) {
+				// If no Java home was returned it means we're using the default
```

Comment:
if we are using the default javac on path - why need to set it ?

Reply:
Basically just to be sure that the program we're about to start will really use the correct JDK. There are many examples of applications using the value of `JAVA_HOME` over what's available in the `PATH`. This way we'd make sure that all values point to the same place.